src/react/hooks/__tests__/useSuspenseQuery.test.tsx

```diff
@@ -5223,6 +5279,45 @@ describe('useSuspenseQuery', () => {
     ]);
   });
 
+  it('suspends when options are set after using `skipToken`', async () => {
```

Comment:
```suggestion
  it('suspends when options change away from `skipToken`', async () => {
```
The test is not really setting any options.

Reply:
Ha you're right, good catch. The equivalent test in `useBackgroundQuery` was even more egregious, so I updated both: [6c861b2](https://github.com/apollographql/apollo-client/pull/11112/commits/6c861b2632fcb86ebfa57559498cf101b2057f60)